src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const personalInfo = sex === 'm'
+    ? `${partnerName} is my wife`
+    : `${partnerName} is my husband`;
```

Comment:
Great, you can also do like this
```suggestion
  const personalInfo = `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}wife`;
```